### DISCUSSION TITLE
Test run for a few good samples

### DISCUSSION THREAD

**Discussion by su-park** [embeddings-benchmark/mteb#1095] [Q&A]
In some evaluations, it can take a long time depending on the environment, 
and errors are sometimes discovered late. 
It would be good to be able to test a small number of cases to see if they work properly.

**Comment by KennethEnevoldsen** [comment]
This seems like a usage question so will transfer it over to discussions

**Reply by KennethEnevoldsen** [reply]
A solution might be to run the benchmark using a smaller model first. However many of the current datasets in MTEB are excessively large. We are working on a lite version of it, which should be a tremendous speed-up.
